index.php

```diff
@@ -0,0 +1,25 @@
+<?php
+$db = new PDO(
+    'mysql:host=DB;dbname=collections',   //DSN
+    'root',                    // Database username
+    'password'                 //Database password
+);
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
You are probably going to need a database connection multiple times for this project. At this stage, I would suggest making a function that you can call to get a DB connection would be a good move. Would also suggest that you put that into it's own separate PHP file.